app/components/UI/Ramp/Deposit/Views/BuildQuote/BuildQuote.test.tsx

```diff
@@ -23,6 +23,27 @@ const mockInteractionManager = {
   runAfterInteractions: jest.fn((callback) => callback()),
 };
 
+const MOCK_US_REGION = {
+  isoCode: 'US',
+  flag: '🇺🇸',
+  name: 'United States',
+  phone: {
+    prefix: '+1',
+    placeholder: '(555) 555-1234',
+    template: '(XXX) XXX-XXXX',
+  },
+  currency: 'USD',
+  recommended: true,
+  supported: true,
+};
+
+const createMockSDKReturn = (isAuthenticated = false, overrides = {}) => ({
+  isAuthenticated,
+  selectedRegion: MOCK_US_REGION,
+  setSelectedRegion: jest.fn(),
+  ...overrides,
+});
```

Comment:
this utility works fine however the references obscure the property we are changing:
```ts
createMockSDKReturn(true)
```
it is not straight forward what `true`. I suggest to avoid the param and use override always:
```suggestion
const createMockSDKReturn = (mockedResult = {}) => ({
  isAuthenticated: false,
  selectedRegion: MOCK_US_REGION,
  setSelectedRegion: jest.fn(),
  ...mockedResult,
});
```

```ts
createMockSDKReturn({ isAuthenticated: true })
```